core/arch/arm/plat-zynq7k/main.c

```diff
@@ -84,7 +84,7 @@ void plat_cpu_reset_late(void)
 		/* primary core */
 #if defined(CFG_BOOT_SECONDARY_REQUEST)
 		/* set secondary entry address and release core */
-		write32(CFG_TEE_LOAD_ADDR, SECONDARY_ENTRY_DROP);
+		write32(TEE_TEXT_VA_START, SECONDARY_ENTRY_DROP);
```

Comment:
This expects to write a physical address, not a virtual address.

Reply:
yes, will fix.